Sources/BluetoothServices/Services/DeviceInformationService.swift

```diff
@@ -60,38 +60,4 @@ public final class DeviceInformationService: BluetoothService, @unchecked Sendab
 
 
     public init() {}
-
```

Comment:
Is it possible to notify these values by default to get updates as they trickle in?

Or is this information expected to be static and therefore not requires this function? Or what was that the reason why this method was removed?

Reply:
These, by spec, are read-only. How and if such values are updated is out of scope (e.g., an updated software version would usually require a restart and a reconnect anyways).

With SpeziBluetooth all characteristics will be supplied with the current value. The method was from before that. So there is currently no need for that method really. That is why it got removed.

Reply:
Sounds great; thanks for the background 🚀